src/components/AssetModels/AssetModelsComponent.jsx

```diff
@@ -94,7 +88,7 @@ export class AssetModelsComponent extends React.Component {
             <Table.Footer>
               <Table.Row>
                 {!_.isEmpty(this.props.assetModels) &&
-                    (
+                  (
```

Comment:
Please move this to the previous line. I.e.
```
{!_.isEmpty(this.props.assetModels) && (
  <Table.HeaderCell colSpan="5" id="pagination-header">
    ....
  </Table.HeaderCell >
)}

Reply:
Noted